Makefile

```diff
@@ -238,7 +238,7 @@ DOWNLOAD_PKG_ARCHIVE = \
         true\
     $(else),\
         mkdir -p '$(PKG_DIR)' && ( \
-            $(WGET) -T 30 -t 3 -O- '$($(1)_URL)' \
+            $(WGET) --content-disposition -T 30 -t 3 -O- '$($(1)_URL)' \
```

Comment:
Why is it needed?

From `man wget`:

> This option is useful for some file-downloading CGI programs that use "Content-Disposition" headers to describe what the name of a downloaded file should be.

Our downloading code does not use filename provided by a server.